mirrord/layer/src/file/filter/read_local_by_default.rs

```diff
@@ -81,10 +81,10 @@ pub fn regex_set_builder() -> RegexSetBuilder {
     .collect();
 
     if let Ok(cwd) = env::current_dir() {
-        patterns.push(format!("^.*{}.*$", cwd.to_string_lossy()));
+        patterns.push(format!("^{}", cwd.to_string_lossy()));
```

Comment:
Does `env::current_dir` return the path with slash at the end? Maybe there's a bug here we could fix as well. If the path does not contain the slash, we'd match `/some/dir2` with `/some/dir`

Reply:
In my tests it did not include the slash, but I guess if current dir is root, it will.
Also I'm not really sure it will never include a slash in other cases (different OSs, different FSs). How about something like:
```suggestion
        patterns.push(format!("^{}(/|$)", cwd.to_string_lossy().trim_end_matches('/')));
```

Reply:
Yeah, I think this works :+1: